extractor/src/extractors/GoModExtractor.ts

```diff
@@ -48,7 +48,7 @@ export default class GoModExtractor implements Extractor {
 
                 case "require":
                     i++;    // requires on subsequent lines
-                    for (; i < lines.length && lines[i] !== ")"; i++) {
+                    for (; i < lines.length && lines[i].trim() !== ")"; i++) {
```

Comment:
Is there a place we can do this once and cache the result (after all, we are doing it in two places right here)

Reply:
I was thinking of trimming each line before we run the main loop using something like this:
`const lines = content.split(/\n+/g).map(i => i.trim());`